manage_breast_screening/participants/models/symptoms.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+from ...core.models import BaseModel, ReferenceDataModel
+from .appointment import Appointment
+from .participant import Participant
+
+
+class SymptomType(ReferenceDataModel):
+    name = models.CharField(null=False, blank=False)
+
+
+class SymptomSubType(ReferenceDataModel):
+    symptom_id = models.ForeignKey(SymptomType, on_delete=models.CASCADE)
+    name = models.CharField(null=False, blank=False)
+
+
+class SymptomAreas(models.TextChoices):
+    RIGHT_BREAST = ("RIGHT_BREAST", "Right")
+    LEFT_BREAST = ("LEFT_BREAST", "Left")
+    BOTH_BREASTS = ("BOTH_BREASTS", "Both")
+    OTHER = ("OTHER", "Other")
+
+
+class ParticipantReportedSymptom(BaseModel):
+    symptom_type_id = models.ForeignKey(SymptomType, on_delete=models.PROTECT)
+    symptom_sub_type_id = models.ForeignKey(
+        SymptomSubType, on_delete=models.PROTECT, null=True, blank=True
+    )
+    participant_id = models.ForeignKey(Participant, on_delete=models.PROTECT)
+    appointment_id = models.ForeignKey(Appointment, on_delete=models.PROTECT)
+    reported_at = models.DateField()
```

Comment:
this field might be implied by the appointment date, but in the future it could be collected pre-appointment so explicitly setting a date seems like a good idea